charts/ratify/templates/configmap.yaml

```diff
@@ -7,7 +7,9 @@ data:
     {
       "executor": {
         "verificationRequestTimeout": {{ .Values.provider.timeout.validationTimeoutSeconds | int | mul 1000 | add -100 }},
-        "mutationRequestTimeout": {{ .Values.provider.timeout.mutationTimeoutSeconds | int | mul 1000 | add -50 }}
+        "mutationRequestTimeout": {{ .Values.provider.timeout.mutationTimeoutSeconds | int | mul 1000 | add -50 }},
+        "useRegoPolicy": {{ .Values.policyProvider.useRegoPolicy }},
+        "executionMode": {{ .Values.policyProvider.executionMode | quote }}
```

Comment:
Do we intend to have more types of execution modes other than passthrough/no passthrough? I'm wondering if we can just change this to something like `enablePassthrough` bool instead?

Reply:
Synced in the pr meeting, make these 2 fields as feature flags now.